update rancher-monitoring and rancher-logging doc (#384)
harvester/docs: docs/logging/harvester-logging.md

The size of logging data is related to the cluster scale, workload and other factors. `Harvester` does not use persistent storage to store log data inside the cluster. Users need to set up a log server to receive logs accordingly.
:::

The logging feature is now implemented with an addon and is disabled by default in new installations.

> Users can enable/disable the `rancher-logging` [addon](../advanced/addons.md) from the Harvester WebUI after installation.

Reviewer: ```suggestion
Users can enable/disable the `rancher-logging` [addon](../advanced/addons.md) from the Harvester UI after installation.
```

#### From UI

 1. Navigate to the `Harvester > Advanced > Addons > rancher-logging` page.
 2. Under the `Fluentbit` tab, change the resource requests and limits.
 3. Under the `Fluentd` tab, change the resource requests and limits.
 4. Click `Save` on the bottom right of the screen.

![](/img/v1.2/logging/modify-logging-resources-from-addon.png)

:::note

The UI configuration is only visiable when the `rancher-logging` addon is enabled.

:::

#### From CLI

> You can also change the resource configurations from the command line using  `kubectl edit addons.harvesterhci.io -n cattle-logging-system rancher-logging`.

Reviewer: ```suggestion
You can use the following `kubectl` command to change resource configurations for the `rancher-logging` addon: `kubectl edit addons.harvesterhci.io -n cattle-logging-system rancher-logging`.
```

> The resource path and default values are as following.

Reviewer: ```suggestion
The resource path and default values are as follows:
```

![](/img/v1.2/logging/modify-logging-fluent-resources.png)

_Available as of v1.2.0_

 The navigation path is changed to `Harvester > Advanced > Addons > rancher-logging`, other steps are same as above.

> ![](/img/v1.2/logging/modify-logging-resources-from-addon.png)

Reviewer 2: I think we could just replace the above steps with v1.2.0 designs since users can check the v1.1 doc if needed. WDUT @LucasSaintarbor

:::note
The UI configuration is only visiable when `rancher-logging` addon is enabled.
:::

#### From CLI

> You can also change the resource configurations from the command line using `kubectl edit managedchart -n fleet-local rancher-logging` and modifying the relevant files.

Reviewer 2: consider removing this in the v1.2.0 doc

values.fluentbit.resources.requests.memory: 50mi
```

_Available as of v1.2.0_

The command line is `kubectl edit addons.harvesterhci.io -n cattle-logging-system rancher-logging`.

The resource path is as following.

```
apiVersion: harvesterhci.io/v1beta1
kind: Addon
metadata:
  name: rancher-logging
  namespace: cattle-logging-system
spec:
  valuesContent: |
    fluentbit:
      resources:
        limits:
          cpu: 200m
          memory: 200Mi
        requests:
          cpu: 50m
          memory: 50Mi
    fluentd:
      resources:
        limits:
          cpu: 1000m
          memory: 800Mi
        requests:
          cpu: 100m
          memory: 200Mi
```

:::note

> Even when the addon is disabled, the configuration may still be changed, but the effect can only be validated when the addon is enabled again.

Reviewer 2: ```suggestion
Even in cases where the addon is disabled, configuration adjustments can still be made. However, the actual impact of these changes can only take effect once the addon is re-enabled.
```
